stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/ChatDomainImpl.kt

```diff
@@ -182,7 +182,13 @@ internal class ChatDomainImpl internal constructor(
 
     @VisibleForTesting
     val defaultConfig: Config = Config(connectEventsEnabled = true, muteEnabled = true)
-    internal var repos: RepositoryFacade = createNoOpRepos()
+    internal var repos: RepositoryFacade
+        get() = _repos ?: createNoOpRepos()
```

Comment:
This getter seems to create `createNoOpRepos` whenever we access the `repos` property.
Don't we need to initialize the `_repos` backing property?

```kotlin
get() {
    if (_repos == null) {
        _repos = createNoOpRepos()
    }
    return _repos!!
}
```